spring-security/README.md

```diff
@@ -49,7 +49,7 @@ Then it auto-configures beans, that are required to initialize the Spring Boot a
 Auto-configuration class | Description
 ---- | --------
 [HybridAuthorizationAutoConfiguration](/spring-security/src/main/java/com/sap/cloud/security/spring/autoconfig/HybridAuthorizationAutoConfiguration.java) | Creates a converter ([XsuaaTokenAuthorizationConverter](/spring-security/src/main/java/com/sap/cloud/security/spring/token/authentication/XsuaaTokenAuthorizationConverter.java)) that removes the xsuaa application identifier from the scope names to enable local scope checks using [Spring's common built-in expression](https://docs.spring.io/spring-security/site/docs/current/reference/html5/#el-common-built-in) `hasAuthority`.
-[HybridIdentityServicesAutoConfiguration](/spring-security/src/main/java/com/sap/cloud/security/spring/autoconfig/HybridAuthorizationAutoConfiguration.java) | Configures a `JwtDecoder` which is able to decode and validate tokens from Xsuaa and Identity service. Furthermore it registers the `XsuaaServiceConfiguration` and `IdentityServiceConfiguration` classes, that gets configured with `xsuaa.*` and `identity.*` properties.
+[HybridIdentityServicesAutoConfiguration](/spring-security/src/main/java/com/sap/cloud/security/spring/autoconfig/HybridIdentityServicesAutoConfiguration.java) | Configures a `JwtDecoder` which is able to decode and validate tokens from Xsuaa and Identity service or Identity service alone. Furthermore it registers the `XsuaaServiceConfiguration` and `IdentityServiceConfiguration` or only `IdentityServiceConfiguration` class, that gets configured with `xsuaa.*` and `identity.*` or only `identity.*` properties.
```

Comment:
Furthermore it registers the`IdentityServiceConfiguration` and optionally ` XsuaaServiceConfiguration`, that gets configured with `identity.*` and `xsuaa.*` properties.